diff -up ./absl/synchronization/internal/graphcycles.cc.gcc11 ./absl/synchronization/internal/graphcycles.cc
--- ./absl/synchronization/internal/graphcycles.cc.gcc11	2020-12-19 11:39:31.244272590 -0500
+++ ./absl/synchronization/internal/graphcycles.cc	2020-12-19 11:39:53.069335816 -0500
@@ -37,6 +37,7 @@
 
 #include <algorithm>
 #include <array>
+#include <limits>
 #include "absl/base/internal/hide_ptr.h"
 #include "absl/base/internal/raw_logging.h"
 #include "absl/base/internal/spinlock.h"
