diff -up ./absl/base/internal/sysinfo_test.cc.armv7 ./absl/base/internal/sysinfo_test.cc
--- ./absl/base/internal/sysinfo_test.cc.armv7	2021-06-09 21:54:44.503755718 -0400
+++ ./absl/base/internal/sysinfo_test.cc	2021-06-09 21:55:53.703375569 -0400
@@ -49,7 +49,7 @@ TEST(SysinfoTest, NominalCPUFrequency) {
   // Emscripten doesn't expose it at all.
 #if defined(__linux__) &&                                                  \
         (defined(__aarch64__) || defined(__hppa__) || defined(__mips__) || \
-         defined(__riscv) || defined(__s390x__)) ||                        \
+         defined(__riscv) || defined(__s390x__)) || defined(__arm__) ||    \
     defined(__EMSCRIPTEN__)
   EXPECT_EQ(NominalCPUFrequency(), 1.0)
       << "CPU frequency detection was fixed! Please update unittest.";
